src/Plugin/Commerce/ShippingMethod/AusPost.php

```diff
@@ -227,13 +237,15 @@ public function __construct(
     ConfigureForm $configurationForm,
     ServiceSupport $serviceSupport,
     ServiceDefinitionManager $serviceManager,
-    ClientInterface $client
+    ClientInterface $client,
+    EventDispatcherInterface $eventDispatcher
```

Comment:
`$eventDispatcher` needs to be added to the constructor's docblock.

Reply:
Fixed